packages/vm/tests/api/runBlock.js

```diff
@@ -145,3 +145,32 @@ tape('should run valid block', async (t) => {
 
   t.end()
 })
+
+async function runWithHf(hardfork) {
+  const vm = setupVM({ hardfork: hardfork })
+  const suite = setup(vm)
+
+  const block = new Block(util.rlp.decode(suite.data.blocks[0].rlp))
+
+  await setupPreConditions(suite.vm.stateManager._trie, suite.data)
+
+  let res = await suite.p.runBlock({
+    block,
+    generate: true,
+    skipBlockValidation: true,
+  })
+  return res
+}
+
+tape('should return correct HF receipts', async (t) => {
+  let res = await runWithHf('byzantium')
+  t.equal(res.receipts[0].status, 1, 'should return correct post-Byzantium receipt format')
+  
+  res = await runWithHf('spuriousDragon')
```

Comment:
Were the receipts different before Spurious Dragon? If so, I think this should be added in the test.

Reply:
I guess not, and if so, this would be rather something to add along when we add the older HFs as well.

Reply:
Ah I was a bit confused, this extra receipts thing was indeed introduced in Byzantium, so this test is OK.